src/components/itemMediaInfo/itemMediaInfo.js

```diff
@@ -0,0 +1,167 @@
+            }
+            if (stream.ChannelLayout) {
+                attributes.push(createAttribute(globalize.translate("MediaInfoLayout"), stream.ChannelLayout));
+            }
+            if (stream.Channels) {
+                attributes.push(createAttribute(globalize.translate("MediaInfoChannels"), stream.Channels + " ch"));
+            }
+            if (stream.BitRate && stream.Codec !== "mjpeg") {
+                attributes.push(createAttribute(globalize.translate("MediaInfoBitrate"), (parseInt(stream.BitRate / 1000)) + " kbps"));
+            }
+            if (stream.SampleRate) {
+                attributes.push(createAttribute(globalize.translate("MediaInfoSampleRate"), stream.SampleRate + " Hz"));
+            }
+            if (stream.BitDepth) {
+                attributes.push(createAttribute(globalize.translate("MediaInfoBitDepth"), stream.BitDepth + " bit"));
+            }
+            if (stream.PixelFormat) {
+                attributes.push(createAttribute(globalize.translate("MediaInfoPixelFormat"), stream.PixelFormat));
+            }
+            if (stream.RefFrames) {
+                attributes.push(createAttribute(globalize.translate("MediaInfoRefFrames"), stream.RefFrames));
+            }
+            if (stream.NalLengthSize) {
+                attributes.push(createAttribute("NAL", stream.NalLengthSize));
+            }
+            if (stream.Type !== "Video") {
+                attributes.push(createAttribute(globalize.translate("MediaInfoDefault"), (stream.IsDefault ? "Yes" : "No")));
+            }
+            if (stream.Type === "Subtitle") {
+                attributes.push(createAttribute(globalize.translate("MediaInfoForced"), (stream.IsForced ? "Yes" : "No")));
+                attributes.push(createAttribute(globalize.translate("MediaInfoExternal"), (stream.IsExternal ? "Yes" : "No")));
+            }
+            if (stream.Type === "Video" && version.Timestamp) {
+                attributes.push(createAttribute(globalize.translate("MediaInfoTimestamp"), version.Timestamp));
+            }
+            html += attributes.join("<br/>");
+            html += "</div>";
+        }
+        if (version.Container) {
+            html += '<div><span class="mediaInfoLabel">' + globalize.translate("MediaInfoContainer") + '</span><span class="mediaInfoAttribute">' + version.Container + "</span></div>";
+        }
+        if (version.Formats && version.Formats.length) {
+            //html += "<div><span class="mediaInfoLabel">"+Globalize.translate("MediaInfoFormat")+"</span><span class="mediaInfoAttribute">" + version.Formats.join(",") + "</span></div>";
+        }
+        /*ToDo
+        if (version.Path && version.Protocol !== "Http" && user && user.Policy.IsAdministrator) {
+            html += '<div><span class="mediaInfoLabel">' + globalize.translate("MediaInfoPath") + '</span><span class="mediaInfoAttribute">' + version.Path + "</span></div>";
+        }*/
```

Comment:
Should probably finish the format and path sections before merging this, do you want me to take a look or are you still working on those two?

Reply:
i would like you to look at it since i didn't  figured out how to hide the path form none admin user or we can remove it in its totality.

Reply:
i think we should remove it b/c admin can use metadata editor to get the file path

Reply:
That sounds annoying tbh.

Reply:
I fixed it to only show for admins.